static/src/javascripts/projects/common/modules/sport/score-board.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+import bonzo from 'bonzo';
+import Component from 'common/modules/component';
+import detect from 'lib/detect';
+import $ from 'lib/$';
+
+const getScoreContainerHtml = (context: Object): string => `
+    <div class="score-container">
+        <div class="score__loading ${context.loadingState}">
+            <div class="loading__text">Fetching the scores…</div>
+            <div class="is-updating"></div>
+        </div>
+    </div>
+`;
+
+class ScoreBoard extends Component {
+    constructor(context: Object): void {
+        super(context);
```

Comment:
Is it necessary to pass `context` to the Component's constructor?  The parent constructor doesn't accept any arguments or contain any logic.